Try to clarify label `x:type/coding` better (#152)
exercism/docs: product/tasks.md

The `x:type/<value>` labels describe the type of work the contributor will be engaged in.

| Tag              | Description                                                       |
| ---------------- | ----------------------------------------------------------------- |
| `x:type/ci`      | Work on Continuous Integration (e.g. GitHub Actions workflows)    |

> | `x:type/coding`  | Work on code that is not part of the content (e.g. track tooling) |

Reviewer: ```suggestion
| `x:type/coding`  | Work on code that is not part of the content (e.g. is a track tooling) |
```
Could be understood _"e.g. is NOT a track tooling"_, because `not` has been used before parenthesis.

Author: I don't know, adding a verb inside the parenthesis makes it sound too complicated to me 🤔. IMO anyone who understands what track tooling is will also know that it is not content. With the article it also sounds somehow wrong, is it possible that `track tooling` is uncountable?

Reviewer: Seems you're right about uncountability/article. Regarding a verb, for me it still makes things clearer. I'm not sure I have anything more to add.
```suggestion
| `x:type/coding`  | Work on code that is not part of the content (e.g. is track tooling) |
```

> | `x:type/coding`  | Work involving writing code that is not part of the student-facing content. This is normally tasks associated with production-tooling (e.g. test runners), internal-tooling (generators), or libraries that Exercism maintains. |

Reviewer 2: ```suggestion
| `x:type/coding`  | Write code that is not part of the student-facing content. This is normally tasks associated with production-tooling (e.g. test runners), internal-tooling (generators), or libraries that Exercism maintains. |
```

Reviewer 2: (I realised this fits better with how the others are worded)

Author: (I can't apply suggestion directly via GH because formatting) [`e663676` (#152)](https://github.com/exercism/docs/pull/152/commits/e6636768cc5eeb2ed7d7b1ed2ede2a2b6c87075c)
